web/test/pages/cms.test.tsx

```diff
@@ -13,6 +13,7 @@ describe("cms", () => {
         "utf8"
       );
 
+      // This is tech debt; if you do need to disable typescript please include a comment justifying why.
```

Comment:
I might modify this slightly in all instances. In most use cases, I'm not sure that I would classify this as technical debt.

In this specific case, we're running into an issue where we cannot easily predict a type. We're not doing something "wrong", but it doesn't nicely conform to TypeScript's rules. I would say something similar for usages related to migrations. There's a pretty good reason for disabling this rule in those locations.

```suggestion
    // If you do need to disable typescript please include a comment justifying why.
```

Reply:
I like this note, thanks! I did wanted to convey that this is not preferable, not just "it's okay to do, just add a comment". How about:
> We should try not to do this; if you do need to disable typescript please include a comment justifying why.

?

Reply:
I like that better.